pkg/proxy/endpoints.go

```diff
@@ -197,14 +197,14 @@ type UpdateEndpointMapResult struct {
 	StaleServiceNames []ServicePortName
 	// List of the trigger times for all endpoints objects that changed. It's used to export the
 	// network programming latency.
-	LastChangeTriggerTimes []time.Time
+	LastChangeTriggerTimes map[types.NamespacedName][]time.Time
```

Comment:
This can technically increase memory usage. Did you think about it?

Reply:
That's correct. Assuming there is 5000 keys, 60 bytes each, it'll increase memory by 0.3 MB.

Reply:
OK - please add a comment that if memory usage of kube-proxy will become too much issue, this can be simplified to []time.Time.